web/src/javascript/app.js

```diff
@@ -1,7 +1,10 @@
 import fakeState from "./fakeState";
-import {draw as drawGame, update as updateGame} from "./game";
```

Comment:
I'm not using it right now. I was planning to use it for interpolation, but I won't be doing it right now.